### ISSUE TITLE
[Export] Duplicate entries getting exported while performing export

### ISSUE THREAD

**Issue by ironmaniiith** [collective/plone.importexport#49]
The current implementation duplicates the `MUST_INCLUDED_ATTRIBUTES` while exporting the content from the site.

**Comment by ironmaniiith** [comment]
Elements are getting repeated [here](https://github.com/collective/plone.importexport/blame/master/src/plone/importexport/browser/importexport.py#L192) and [here](https://github.com/collective/plone.importexport/blame/master/src/plone/importexport/browser/importexport.py#L200)

**Comment by ironmaniiith** [comment]
@Shriyanshagro @djay 
In [this](https://github.com/collective/plone.importexport/blame/master/src/plone/importexport/browser/importexport.py#L198) comment, it's written that `MUST_INCLUDED_ATTRIBUTES` must be present in the starting position of the headers. However, I tried exporting the content without it being present in the starting and got no error. I also tried tests but none of them fails. Can anyone tell what could be the possible it being present in the beginning?
